experiments/Atmos_GCM/heldsuarez.jl

```diff
@@ -21,17 +21,19 @@ function init_heldsuarez!(bl, state, aux, coords, t)
 
     FT = eltype(state)
 
-    # TODO: change to altitude function
-    r = norm(coords, 2)
-    h = r - FT(planet_radius)
-    # h = altitude(bl.orientation, aux)
+    h = altitude(bl.orientation, aux)
```

Comment:
I’d prefer to use `z` for altitude (it’s what we usually do; we use `h` for enthalpy of sometimes a fixed height)